lib/index.js

```diff
@@ -14,6 +14,7 @@ import fs from 'fs'
  * @param {String} [config.iife] Optional. Specify filename to write Self executing <script>, exposing all icons as symbols on page,
  * @param {String} [config.dts] Optional. Specify filename to write TypeScript definitions with `export declare const`
  * @param {String} [config.dtsx] Optional. Specify filename to write TypeScript definitions for JSX with `export declare const`
+ * @param {includeBanner: Boolean, parser: Function, filename: String}[]} extensions Optional. List of extensions to generate custom outputs
```

Comment:
Missing a bracket and syntax to be part of config. Should also wrap object in array notation for easier legibility and put optional key last.
I suggest an update to the following: `@param {[{parser: Function, filename: String, includeBanner?: Boolean}]} [config.extensions]`

Reply:
Thank you! JSDoc is a new stuff for me